db-init/src/main/resources/database/migrations/V4.3__Update_Claim_Table.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE claim 
+ADD COLUMN vbms_id VARCHAR, -- uuid?
+ADD COLUMN off_ramp_reason VARCHAR, 
+ADD COLUMN presumptive_flag BOOLEAN, 
+ADD COLUMN disability_action_type VARCHAR,
+ADD COLUMN in_scope BOOLEAN,
+ADD COLUMN submission_source VARCHAR,
```

Comment:
What is `submission_source`? I don't see it defined in the wiki.

Reply:
All columns should have a comment to describe the field (bonus points if the comment provides sample values, particularly for VARCHAR that are enumerations in practice) -- https://www.postgresql.org/docs/current/sql-comment.html.